tests/functional/cylc-install/07-no-recursive-installations.t

```diff
@@ -74,14 +67,8 @@ run_fail "${TEST_NAME_BASE}-Nth-parent" cylc install -C "$PWD" --flow-name "${TE
 grep_ok "WorkflowFilesError.*exists" "${TEST_NAME_BASE}-Nth-parent.stderr"
 
 
-TEST_NAME="${TEST_NAME_BASE}-run-dir-sibling"
-TEST_FOLDER=cylctb-$(uuidgen)
-TEST_FOLDERS+=("$TEST_FOLDER")
-cylc install -C "$PWD" --flow-name "${TEST_FOLDER}/bar"
-run_fail "${TEST_NAME}" cylc install -C "$PWD" --flow-name "${TEST_FOLDER}"
-grep_ok "Nested install directories not allowed" "${TEST_NAME}.stderr"
-
```

Comment:
I don't think this test should be deleted. If I'm not mistaken it's the same as https://github.com/cylc/cylc-flow/pull/4442#issuecomment-954637047, which is (incorrectly) not raising an error unfortunately